### PR TITLE
Projects view

### PR DESCRIPTION
**Pull request by jmay4826**

### CODE REVIEW

File: server/api/projects/projectsCtrl.js [hunk 1]
```diff
@@ -1,7 +1,38 @@
...
+    db.projects.getProjectsByCohort({ cohort_id }).then(result => {
+      const projects = {};
+      groupById(result, projects, 'total_incomplete');
+      groupRowData(result, projects, 'projects');
+
+      // group by student
+
+      res.json(objToArray(projects));
+    });
     console.log(req.body, req.params);
```
Comment (SheaClose) [inline server/api/projects/projectsCtrl.js]: This necessary?

File: server/api/projects/projectsCtrl.js [hunk 2]
```diff
@@ -1,7 +1,38 @@
...
   },
+  updateCompletion(req, res) {
+    const db = req.app.get('db');
+    const { id } = req.params;
+    const { completion, cohort_id } = req.body;
+    console.log(id, completion);
```
Comment (SheaClose) [inline server/api/projects/projectsCtrl.js]: or here?

File: server/api/projects/projectsCtrl.js [hunk 3]
```diff
@@ -1,7 +1,38 @@
...
+    db.projects
+      .updateCompletion({ id, completion })
+      .then(() => db.projects.getProjectsByCohort({ cohort_id }))
+      .then(result => {
+        const projects = {};
+        groupById(result, projects, 'total_incomplete');
+        groupRowData(result, projects, 'projects');
+        console.log(projects);
```
Comment (SheaClose) [inline server/api/projects/projectsCtrl.js]: or here?

File: server/api/students/studentCtrl.js [hunk 4]
```diff
@@ -173,6 +158,7 @@ module.exports = {
   },
   addOneOnOne(req, res) {
     const db = req.app.get('db');
+    console.log(req.body);
```
Comment (SheaClose) [inline server/api/students/studentCtrl.js]: or here?

**Comment by jmay4826** [comment]
Nope :) Removed them all

**Pull request merged**